src/main/java/blue/Blue.java

```diff
@@ -0,0 +1,41 @@
+package blue;
+
+import blue.task.TaskList;
+
+/**
+ * The main class for the Blue application, which manages tasks using a {@link TaskList} and a {@link Parser}.
+ * The application starts by greeting the user, then parsing user input to manage tasks, and finally saying farewell.
+ */
+public class Blue {
+    /** The task list managed by the application. */
+    private TaskList tasklist;
+
+    /** The parser that handles user input. */
+    private Parser parser;
```

Comment:
The name parser is well-chosen and follows Java naming conventions. I like how the variable name is clear and indicates its purpose.